networking/synthetic/stac-prepare-nodes.yaml

```diff
@@ -2,7 +2,39 @@
 - hosts: stac_nodes
   remote_user: root
   tasks:
-    
+
+  - name: install epel 7
+    yum:
+      name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+
+  - name: add rhel latest repos
+    yum_repository:
+      name: rhel_latest
+      description: taken form https://github.com/redhat-performance/perf-dept/blob/master/repo_files/rhel7-latest.repo
```

Comment:
what typo for the name?
well, im not sure we have to say more than that for a description ?! it is all about extending the repos